recipes/dis3/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dis3" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "ee9f71e1a0c01a3ba742a36be91147948db99da3ee29094dccb48857c54608ef" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py>=30]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - dis3
+
+about:
+  home: https://pypi.python.org/pypi/dis3
+  license: MIT
+  license_family: MIT
```

Comment:
There is no license file and no way to fix that. So we should include a note to that effect here.

Reply:
note: Done below. Thanks. :)